pkgs/development/python-modules/convertdate/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pymeeus, pytz }:
+
+buildPythonPackage rec {
+  pname = "convertdate";
+  version = "2.2.0";
+
+  # Tests are not available in the PyPI tarball so use GitHub instead.
+  src = fetchFromGitHub {
+    owner = "fitnr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04j8k7a9qndmawy3m345py74y18hw7lb6gc0qp0mr8d68x99xjq0";
+  };
+
+  propagatedBuildInputs = [ pymeeus pytz ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/fitnr/convertdate;
```

Comment:
```suggestion
    homepage = "https://github.com/fitnr/convertdate";
```

Reply:
Why the double quotes? There are many examples in the manual without the double quotes: https://nixos.org/nixpkgs/manual/#chap-reviewing-contributions

Reply:
Got an answer from IRC. Thanks, fixed.